test/Listener/DefaultListenerAggregateTest.php

```diff
@@ -62,6 +62,7 @@ public function testDefaultListenerAggregateCanAttachItself()
                 'Zend\ModuleManager\Listener\OnBootstrapListener',
                 'Zend\ModuleManager\Listener\ConfigListener',
                 'Zend\ModuleManager\Listener\LocatorRegistrationListener',
+                'Zend\ModuleManager\ModuleManager',
```

Comment:
Shouldn't this already be registered (due to registration of the `onLoadModules` listener)? Why is this change necessary?